src/main/java/org/gridsuite/modification/server/dto/ByFilterDeletionInfos.java

```diff
@@ -0,0 +1,68 @@
+/*
+  Copyright (c) 2023, RTE (http://www.rte-france.com)
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.powsybl.commons.reporter.Reporter;
+import com.powsybl.commons.reporter.ReporterModel;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+import org.gridsuite.modification.server.dto.annotation.ModificationErrorTypeName;
+import org.gridsuite.modification.server.entities.equipment.deletion.ByFilterDeletionEntity;
+import org.gridsuite.modification.server.modifications.AbstractModification;
+import org.gridsuite.modification.server.modifications.ByFilterDeletion;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author Antoine Bouhours <antoine.bouhours at rte-france.com>
+ */
+@SuperBuilder
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString(callSuper = true)
+@Schema(description = "By filter deletion")
+@JsonTypeName("BY_FILTER_DELETION")
+@ModificationErrorTypeName("BY_FILTER_DELETION_ERROR")
+public class ByFilterDeletionInfos extends ModificationInfos {
+    @Schema(description = "Equipment type")
+    private String equipmentType;
```

Comment:
As discussed on Slack, even if it's done like that for `EquipmentDeletion`, I think we should use an enum.
Maybe in a other PR to mutualize with the existing code ?

Reply:
I used an enum for this PR as the code does not depend on `EquipmentDeletion`. I updated  `EquipmentDeletion` in https://github.com/gridsuite/network-modification-server/pull/400